djangocms_attributes_field/fields.py

```diff
@@ -85,6 +86,18 @@ def to_str(cls, obj, field_name):
                 attrs.append('{key}="{value}"'.format(key=key, value=conditional_escape(val)))
         return mark_safe(" ".join(attrs))
 
+    def contribute_to_class(self, cls, name, **kwargs):
+        """
+        Adds a @property: «name»_str that returns a string representation of
+        the attributes ready for inclusion on an HTML element.
+        """
+        super(AttributesField, self).contribute_to_class(cls, name, **kwargs)
+        # Make sure we're not going to clobber something that already exists.
+        property_name = '{name}_str'.format(name=name)
+        if not hasattr(cls, property_name):
```

Comment:
Yea, tricky. However, its only a convenience property and its exceedingly simple to create your own if this one collides with something else.